new/LIT6513Nagar.xml

```diff
@@ -57,6 +58,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <listRelation>
                     <relation name="saws:isAttributedToAuthor" active="LIT6513Nagar" passive="PRS7104Minas"/>
                 </listRelation>
```

Comment:
add the about relation to the keyword "Masqal" (feast) and possibly also "TrueCross" (the actual cross found)

Reply:
Can you please explain, keywords, do you mean this: https://betamasaheft.eu/authority-files/TrueCross/main? But there's no record number here.

Reply:
You do not need the record number, simply the keyword (passive="Masqal", passive="TrueCross")